app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -321,10 +325,18 @@ private void setListeners() {
             controller.initiateCameraPick(getActivity(), inAppCameraLocationPermissionLauncher);
             animateFAB(isFabOpen);
         });
+        binding.fabCamera.setOnLongClickListener(view -> {
+            ViewUtil.showShortToast(getContext(),R.string.add_contribution_from_camera);
```

Comment:
I'm not sure this is true - this isn't just to add contributions from the camera (e.g. you can add it from the gallery).

Reply:
This is for a fab camera, one you can see below
![Screenshot_2024-02-13-10-48-59-799_fr.free.nrw.commons.beta.jpg](https://github.com/commons-app/apps-android-commons/assets/126143257/be605594-7d59-4305-bd32-df8d294d60d2)

The first one of the three, doesn't  the first one only opens camera ?

![Screenshot_2024-02-13-10-51-29-180_com.android.camera.jpg](https://github.com/commons-app/apps-android-commons/assets/126143257/d247375a-a461-44e1-8ce4-8e4f86470e4f)


On my device this opens, there is not a way to open gallery from here.

Reply:
Apologies my bad, you're right!